kelp/fast.pyx

```diff
@@ -67,26 +67,745 @@ cdef float H(int l, float theta, float alpha) nogil:
     if l == 0:
         return 1
     elif l == 1:
-        return 2 * tilda_mu(theta, alpha)
+        return 2*tilda_mu(theta, alpha)
     elif l == 2:
-        return 4 * tilda_mu(theta, alpha) ** 2 - 2
+        return 4*tilda_mu(theta, alpha)**2 - 2
     elif l == 3:
-        return 8 * tilda_mu(theta, alpha) ** 3 - 12 * tilda_mu(theta, alpha)
+        return 8*tilda_mu(theta, alpha)**3 - 12*tilda_mu(theta, alpha)
     elif l == 4:
-        return (16 * tilda_mu(theta, alpha) ** 4 - 48 *
-                tilda_mu(theta, alpha) ** 2 + 12)
+        return 16*tilda_mu(theta, alpha)**4 - 48*tilda_mu(theta, alpha)**2 + 12
     elif l == 5:
-        return (32 * tilda_mu(theta, alpha) ** 5 - 160 *
-                tilda_mu(theta, alpha) ** 3 + 120)
```

Comment:
The missing `tilda_mu` call in the last term in this line was causing a bug in all previous versions (!), though it only mattered for lmax=5.